easybuild/framework/easyconfig/tools.py

```diff
@@ -538,6 +539,28 @@ def review_pr(paths=None, pr=None, colored=True, branch='develop'):
         else:
             lines.extend(['', "(no related easyconfigs found for %s)\n" % os.path.basename(ec['spec'])])
 
+    labels = []
+    file_info = det_file_info(pr_files, download_repo_path)
+    if any(file_info['new_folder']):
+        labels.append('new')
+    if any(file_info['new_file_in_existing_folder']):
+        labels.append('update')
+
+    github_account = build_option('pr_target_account')
+    github_repo = build_option('pr_target_repo')
+    github_user = build_option('github_user')
+    pr_data, _ = fetch_pr_data(pr, github_account, github_repo, github_user)
+    pr_labels = [label['name'] for label in pr_data['labels']]
+
+    missing_labels = []
+    for label in labels:
+        if label not in pr_labels:
+            missing_labels.append(label)
+
+    if missing_labels:
+        if not post_pr_labels(pr, labels):
```

Comment:
I wouldn't let `--review-pr` add labels automagically, I feel this should remain a read-only option, as you mentioned.

But we can add support for a separate `--add-pr-labels` option?

Reply:
reverted adding labels. It's probably best to leave `--add-pr-labels` to a separate PR then